setup.py

```diff
@@ -6,15 +8,14 @@
     name="async_generator",
     version=__version__,
     description="Async generators for Python 3.5",
-    long_description=open("README.rst").read(),
+    long_description=Path(__file__).with_name("README.rst").read_text('utf-8'),
```

Comment:
Figures out the path to `setup.py`, even if it's not in the current directory, and then changes the file name of the path to `README.rst` and finally reads the file contents as UTF-8 (the explicit encoding is necessary on Anaconda installs [like on RTD] where str.decode() for some incomprehensible reason does not default to utf-8!).
In the early days of my most popular project, APScheduler, I got a PR from someone who was doing RPM packaging of APScheduler. They were complaining that apscheduler's setup.py assumes that the working directory is the project directory, which apparently is not always the case. This line avoids that by not assuming anything about the CWD (and it closes the properly too, not that it really matters in the case of setup.py).